src/components/UiDebioBanner.vue

```diff
@@ -17,13 +17,15 @@
               ui-debio-icon.banner__decoration(
                 v-if="withDecoration"
                 :icon="particleIllustration"
-                size="470"
+                width="350"
+                height="270"
+                color="red"
                 view-box="0 0 390 235"
               )
 </template>
 
 <script>
-import { particleIllustration } from "@debionetwork/ui-icons/src/particleIllustration"
+import { particleIllustration } from "@debionetwork/ui-icons"
```

Comment:
Please undo these changes @bukandicki it will increase build size.

Reply:
okai @agustinustheo 👍🏼